packages/cursorless-engine/src/CommandHistoryAnalyzer.ts

```diff
@@ -5,26 +5,31 @@ import {
   ScopeType,
 } from "@cursorless/common";
 import globRaw from "glob";
-import { readFile } from "node:fs/promises";
-import path from "node:path";
 import { promisify } from "node:util";
 import { canonicalizeAndValidateCommand } from "./core/commandVersionUpgrades/canonicalizeAndValidateCommand";
 import { ide } from "./singletons/ide.singleton";
 import { getPartialTargetDescriptors } from "./util/getPartialTargetDescriptors";
 import { getPartialPrimitiveTargets } from "./util/getPrimitiveTargets";
 import { getScopeType } from "./util/getScopeType";
+import { generateCommandHistoryEntries } from "./generateCommandHistoryEntries";
+import groupby from "./groupby";
+import { mapAsync } from "./mapAsync";
+import { asyncIteratorToList } from "./asyncIteratorToList";
 
-const glob = promisify(globRaw);
+export const glob = promisify(globRaw);
```

Comment:
This feels like a weird export to have in this file. Should we just move this to a common library instead?

Reply:
Nice catch. Happened during auto refactor